docs/install.md

```diff
@@ -40,7 +40,7 @@ Follow the steps below to install Codacy on an existing Kubernetes cluster using
     helm repo update
     helm upgrade --install codacy codacy-stable/codacy \
       --namespace codacy \
-      --recreate-pods \
+      --atomic \
```

Comment:
we need to set a higher timeout as well here... I guess ...

Reply:
true. Does adding a timeout of 5 min sounds reasonable?